test/blackbox/common/mesh_event_handler.h

```diff
@@ -90,6 +90,20 @@ struct cond_flag {
 	bool flag;
 };
 
+typedef struct {
+	mesh_event_t  currentEv;
+	mesh_event_t  nextEv;
+} meshlink_node_status_lookup_t;
+
+typedef struct {
+	uint16_t                        node_id;
+	mesh_event_t                    currentEv;
+	meshlink_node_status_lookup_t   *lookup;
+	uint16_t                        current_index;
+	uint8_t                         max_events;
+	bool                            status;
```

Comment:
You don't need `status` either, since it's basically equivalent to `current_index == max_events`.